portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsDateField.testcase

```diff
@@ -495,7 +495,7 @@ definition {
 
 		Form.viewFieldReference(
 			fieldName = "Date",
-			fieldValue = "Date1234");
+			key_fieldReference = "Date1234");
```

Comment:
@MoryAlvesBR, please rename to `fieldReference`. On macro `viewFieldReference`, please make something like:
```
key_fieldReference = "${fieldReference}";
```
Also, please update all calls to this macro to reflect this change. At this moment, it's being used on 2 places only. :wink: 

Thanks in advance